packages/buildessential.rb

```diff
@@ -8,9 +8,11 @@ class Buildessential < Package
 
   is_fake
 
+
+  #use a newer version of bash
+  depends_on 'bash'
```

Comment:
This should probably be optional.  Feel free to argue why it should be included as a core package.